src/net_processing.cpp

```diff
@@ -4394,9 +4394,20 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
             !pto->HasPermission(PF_FORCERELAY)) {
             CAmount currentFilter = m_mempool.GetMinFee(gArgs.GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000).GetFeePerK();
             int64_t timeNow = GetTimeMicros();
+            static FeeFilterRounder filterRounder{CFeeRate{DEFAULT_MIN_RELAY_TX_FEE}};
```

Comment:
Nit: as you're touching all places where this variable is used, convert it to current naming style?

Also: can this be `const`?

Reply:
`filterRounder` has a fast random context, which is not const. Calling `round` on it will change the state.

Reply:
Uh, that means it needs a mutex (or very clearly stated assumptions that no two PeerLogicValidation objects can exist that could be accessed from separate threads).

Reply:
This seems unrelated to the changes here. Filed under a new issue: #19254

Reply:
Cool, thanks.